apps/condo/domains/document/access/DocumentCategory.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * Generated by `createschema document.DocumentCategory 'name:Text; order:Integer;'`
+ */
+
+const { throwAuthenticationError } = require('@open-condo/keystone/apolloErrorFormatter')
+
+async function canReadDocumentCategories ({ authentication: { item: user } }) {
+    if (!user) return throwAuthenticationError()
+    if (user.deletedAt) return false
+
+    return {}
+}
+
+async function canManageDocumentCategories ({ authentication: { item: user }, operation }) {
+    if (!user) return throwAuthenticationError()
+    if (user.deletedAt) return false
+    if (user.isAdmin) return true
+
+    if (user.isSupport && (operation === 'create' || operation === 'update')) {
```

Comment:
Why you cannot just use  (user.isAdmin || user.isSupport) return true? Are you using this util outside of create/update access operations?

Reply:
We also have `delete` operation, but ok, changed to 
```
if (user.isAdmin || user.isSupport) return true
```

Reply:
Remove edit access for supports